src/plugins/intel_gna/src/backend/am_intel_dnn.hpp

```diff
@@ -272,22 +272,22 @@ class AMIntelDNN {
     }
 
     template <class T>
-    void AdvanceOperationIfAllApplied(const std::vector<intel_dnn_component_t>& cmp, int i, T*& operation) {
+    void AdvanceOperationIfAllApplied(const std::vector<intel_dnn_component_t>& cmp, size_t i, T*& operation) {
         if (i == cmp.size() - 1 || cmp[i + 1].operation != kDnnPiecewiselinearOp) {
```

Comment:
I suggested change to `i + 1 = cma.size()` becayse I wanted to avoid situation that cmp.size() will be 0 so substracting 1 would be undefined behaviour, but I understand it is not possible here.

Reply:
If `cmp.size() == 0` we'd have an out-of-bounds error anyway, in the next condition. I could add some check for correctness of the data, but:
- it has worked before
- it's legacy code anyway

do you think it's in scope of this task?